src/components/Photo/DocumentLiveCapture.js

```diff
@@ -81,15 +81,13 @@ export default class DocumentLiveCapture extends Component<Props, State> {
     const { hasBecomeInactive, hasCameraError, isCapturing } = this.state
     const id1SizeDocuments = new Set([ 'driving_licence', 'national_identity_card' ])
     const documentSize = id1SizeDocuments.has(documentType) ? 'id1Card' : 'id3Card'
-    const idealCameraHeightInPixels = 1280
     return (
       <div className={style.container}>
         {this.state.isCapturing ? (
           <Spinner />
         ) : (
           <Camera
-            facing={{ exact: 'environment' }}
-            idealCameraHeight={idealCameraHeightInPixels}
```

Comment:
I guess this means we can only ever get 720p quality images at best in order to work across all/most devices' cameras. Not sure if it'll be a major issue since Doc Live Capture was intended to enable capturing higher quality images. 🤔

Reply:
720 is usually good enough in terms of image quality and it works across many more devices than higher resolutions do. That's why we made it the default value. I remember pointing out during the implementation of the feature that using 1280 might break compatibility for many devices because we did some benchmarking in the past that lead us to use 720. This fail silently so it's not easy to track either

Reply:
Anyways, that's a good point. Maybe we can see what the impact is on IQ from the time we release this feature and if we see a big drop we can consider increasing the resolution